pyzx/congruences.py

```diff
@@ -0,0 +1,153 @@
+import numpy as np
```

Comment:
Missing the standard header on top of the file. Also should have a top level documentation comment.

Reply:
Also, there is no mypy annotation here, which I guess is fine for now, but I would like pyzx to have annotations at least for all the function signatures. All the functions are also missing documentation strings.